packages/vm/src/modules/in_memory_cache.rs

```diff
@@ -113,7 +113,7 @@ mod tests {
         // Module and Store are aligned (https://doc.rust-lang.org/reference/type-layout.html#the-default-representation).
         // So we get this value by trial and error. It can change over time and across platforms.
         let value_size = mem::size_of::<Module>();
-        assert_eq!(value_size, 48);
+        assert_eq!(value_size, 56);
```

Comment:
Could you check if the comment above is still up-to-date after the size changed?

Reply:
There's one more `Arc`. Fixed!